internal/serve/httphandler/profile_handler.go

```diff
@@ -177,13 +177,10 @@ func (h ProfileHandler) PatchOrganizationProfile(rw http.ResponseWriter, req *ht
 	}
 	var nonEmptyChanges []string
 	for k, v := range requestDict {
-		if !utils.IsEmpty(v) {
-			value := v
-			if k == "Logo" {
-				value = "..."
-			}
-			nonEmptyChanges = append(nonEmptyChanges, fmt.Sprintf("%s='%v'", k, value))
+		if k == "Logo" {
+			v = "..."
 		}
+		nonEmptyChanges = append(nonEmptyChanges, fmt.Sprintf("%s='%v'", k, v))
```

Comment:
❓Was that non-empty check  `!utils.IsEmpty(v)` removed on purpose here ? If yes should we rename this variable to reflect that ?

Reply:
Yes, it was removed because it was redundant. The serialized struct has `omitempty` in all its fields, so the empty fields were already filtered off during serialization.

The variable name is still accurate.